extensions.json

```diff
@@ -792,6 +792,18 @@
       "version": "3.0.6",
       "checkout": "v3.0.6"
     },
+    {
+      "id": "ms-vscode.cpptools",
+      "repository": "https://github.com/Microsoft/vscode-cpptools",
```

Comment:
The `cpptools` extension has a license that makes it incompatible with anything except Microsoft products:

https://github.com/microsoft/vscode-cpptools/blob/master/RuntimeLicenses/cpptools-LICENSE.txt

Please remove that extension.